src/api/api.tsx

```diff
@@ -126,6 +126,12 @@ export function uppyOnAfterResponse(xhr: XMLHttpRequest) {
           else resolve();
         });
       });
+    } else if (retryCount === 3 || (xhr.status >= 400 && xhr.status < 600)) {
+      let errorMessage = (JSON.parse(xhr.responseText) as APIError).detail;
+      if (errorMessage.includes('Limit for the maximum number of images')) {
```

Comment:
i wonder if it could do with a comment here to explain why attachments aren't handled in this file?